spec/rubocop/cop/style/negated_if_else_condition_spec.rb

```diff
@@ -129,6 +129,33 @@
     RUBY
   end
 
+  it 'moves comments to correct branches during autocorrect' do
+    expect_offense(<<~RUBY)
+      if !condition.nil?
+      ^^^^^^^^^^^^^^^^^^ Invert the negated condition and swap the if-else branches.
+        # part B
+        # and foo is 39
+        foo = 39
```

Comment:
It seems that a syntax error occurs due to auto-correction when there are multiple statements as shown below.

```diff
        # part B        
        # and foo is 39
        foo = 39
+       foo = 40
```

Also, the following case does not seem to be corrected as expected.

```diff
        # part B        
        # and foo is 39
        foo = 39
+       # and foo is 40
+       foo = 40
```